tests/unit/test_configuration.py

```diff
@@ -469,3 +469,34 @@ def test_no_noise():
             column_noise_dict = dataset_column_dict[column]
             for column_noise_type in column_noise_dict.keys():
                 assert column_noise_dict[column_noise_type][Keys.CELL_PROBABILITY] == 0.0
+
+
+@pytest.mark.parametrize(
+    "dataset_name",
+    [
+        DATASETS.census.name,
```

Comment:
would it make sense to iterate over DATASETS here, or is it important to be explicit about the named datasets that are included?

Reply:
It would I jus thaven't taken the time to convert all of the parameterizations. I initially wrote it like that to more easily allow for skipping tests that were unimplemented and so we could more easily see when stuff was missing. Now that all datasets have been implemented it would be better to just iterate.